src/app/index.ts

```diff
@@ -359,56 +359,43 @@ module.exports = class extends yo {
   }
 
   _postInstallHints(): void {
+    let projFolder: string = /\s/.test(this._destinationRoot) ? "\"" + this._destinationRoot + "\"" : this._destinationRoot;
```

Comment:
It is kinda hard to understand regex on their own.
Can you add some commentary explaining what this line does?